core/src/main/java/org/mule/runtime/core/api/policy/PolicyChain.java

```diff
@@ -129,6 +129,10 @@ public CoreEvent process(CoreEvent event) throws MuleException {
         .doOnNext(e -> notificationHelper.fireNotification(e, null, PROCESS_END));
   }
 
+  public MuleContext getMuleContext() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "MuleContext"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7763&issues=AWnLHoQ9otIXnmsUntJ_&open=AWnLHoQ9otIXnmsUntJ_)

[ISSUE_KEY]: AWnLHoQ9otIXnmsUntJ_